backend/internal/models/user.go

```diff
@@ -6,6 +6,7 @@ type User struct {
 	Name       string `json:"name"`
 	Email      string `json:"email"`
 	InSanDiego bool   `json:"in_san_diego"`
+	Favorites  []Book `json:"favorite_books"`
```

Comment:
Let's not do this, this is the model for GET /users/:id. We don't need the favorites array anywhere other than the favorites API